### DISCUSSION TITLE
ERROR： There was an error committing your changes: File could not be edited.

### DISCUSSION THREAD

**Discussion by levenwindy** [community/community#54940] [Actions]
### Select Topic Area

Question

### Body

Commit changes...

![Commit changes...](https://s2.loli.net/2023/05/09/vN67j12hbcPAJQf.png)

Delete file

![Delete file](https://s2.loli.net/2023/05/09/1hSHPpJTVFgEvxe.png)

**Comment by imightknoiw** [comment]
This is an active ongoing issue. Please check [https://www.githubstatus.com/](https://www.githubstatus.com/) for more updates.

**Comment by Akash1134** [comment]
Thanks for posting in the GitHub community and we appreciate your patience. You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

Current Status: Actions is experiencing degraded performance. We are still investigating and will provide an update when we have one.

**Comment by Hisan-007** [comment]
**Solution in my case:** 
I'm not sure if that's the solution for all possible cases, but in my cases, it was happening because i was creating another file with the same name, not realizing there was a file already with the same name. for instance i had `datastructure.md` and i was creating another file `datastructure.md`. So the solution was simply to create file with a different name, as it was a mistake.

**Reply by Suwija** [reply]
work for me, thanks

**Comment by mikey-liu** [comment]
same problem with me    .   because i  am using the network in company?

**Reply by KosmosLV** [reply]
I have this exact scenario. It is likely that that is the case.

**Comment by bryanbocao** [comment]
Same problem just now and I couldn't even comment. It said "You can't comment at this time.".

**Comment by RajagopalanV** [comment]
Is this resolved? I am getting "There was an error committing your changes: File could not be created" when I change or creating new file

**Reply by zC4sTr0** [reply]
Today I started getting the same error

**Comment by r-kala** [comment]
I am facing the same issue, the page did not load first with server error 500. Now it loaded but I am unable to make any changes. Was it resolved for anyone and if it did how long did it take?

**Comment by bryanbocao** [comment]
@RajagopalanV It seemed solved after a few minutes to hours when I commented it. It looked like some intermediate server issues in Github in general and later the website went back normally.

**Comment by zC4sTr0** [comment]
I'm getting this error now and github status is showing operational:

![image](https://github.com/community/community/assets/38432614/eb2d630e-3e7d-4de5-a3db-6fbf07d4d77d)

**Comment by phoppermann** [comment]
In my case, editing a file and creating a PR in one step didn't work. Creating the branch manually and committing directly to it worked.

**Comment by zhiyi-zhang-duke** [comment]
This is still not working in 2026. It's especially bad because you can't copy code from a suggestion.

**Comment by Tsarros202** [comment]
A bit naive on my part, but I ran into this error after trying to commit to a repository I no longer had access to. I had switched accounts in another browser tab, the page had not refreshed and the repository was still visible.

**Comment by weskerty** [comment]
This happens to me when trying to delete a GitHub action with the .OFF extension.
